packages/sdk-v2/src/sdk/goodid/types.ts

```diff
@@ -0,0 +1,30 @@
+// Reference: https://veramo.io/docs/basics/verifiable_data
+export interface CredentialProof {
+  type: string;
+  jwt: string;
+}
+
+export interface CredentialSubject {
+  id: string;
+  [key: string]: any;
+}
+
+export interface Issuer {
+  id: string;
+}
+
+export interface VerifiableCredential {
```

Comment:
it's certificate (in our tasks definition)

Reply:
But its credential in code: https://github.com/GoodDollar/GoodServer/blob/4ee268df5abb97eed05f6bf86031be4a30ea0c3f/src/server/goodid/goodid-middleware.js#L118

an interface for Certificate would be as I see it
```
interface Certificate {
   credential: VerifiableCredential
}
```

Should it not read as: A certificate can have multiple credentials?
the interface VerifiableCredential is about a single credential

Reply:
I even think this
https://github.com/GoodDollar/GoodServer/blob/4ee268df5abb97eed05f6bf86031be4a30ea0c3f/src/server/goodid/utils.js#L84
will fail if we nest a credential within 'certificate'

IE:
(Based on veramo simple demo)
![image](https://github.com/GoodDollar/GoodWeb3-Mono/assets/6606028/0bbdce60-1c44-4adf-a398-931f661f438e)

meaning I believe its at least missing a destructure here: https://github.com/GoodDollar/GoodServer/blob/4ee268df5abb97eed05f6bf86031be4a30ea0c3f/src/server/goodid/goodid-middleware.js#L149